Userland/Libraries/LibJS/Forward.h

```diff
@@ -163,6 +163,11 @@ class TypedArrayPrototype;
 // Tag type used to differentiate between u8 as used by Uint8Array and u8 as used by Uint8ClampedArray.
 struct ClampedU8;
 
+enum class AllowSideEffects {
+    Yes,
+    No
+};
```

Comment:
I don't think this belongs in the forward header - not sure where else it should go either :thinking:

Reply:
I originally put it in the Object header, but the IndexedProperties header cant include that one (due to the cyclic dependency) so that wont work, and putting it in the IndexedProperties header feels wrong :/